integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -59,8 +61,7 @@ const testRunWorkflowOnSnapshotFn = _.flow(
   await fillInReplace(page, input({ labelContains: 'echo_to_file out attribute' }), 'workspace.result')
 
   await click(page, clickable({ text: 'Save' }))
-  // Wait for "Run Analysis" to become enabled.
-  await page.waitForXPath(enabledClickable({ text: 'Run analysis' }))
+  await clickableEnabled(page, { text: 'Run analysis' })
```

Comment:
The reason I didn't put it in the launch helper function is because in the other test, `run-workflow`, the button should never be disabled. 

My slight preference is to go with `await findElement(page, clickable({text: 'Run analysis', isDisabled: false}))` or keep the separate method… but I don't have very strong feelings as long as there is a way to indicate that we are expecting the button to be initially disabled.

Reply:
I still feel that, 99% of the time, asking for something `clickable` is because you want to `click` it. Therefore, `isDisabled: false` should be the default.

I don't see a lot of value in worrying about whether or not the button is or was disabled here. This test is to assert that a person will be able to successfully run a workflow against a snapshot. It is not to assert that, when running a workflow against a snapshot, the button goes from disabled to enabled (and that it's never disabled when running a workflow against entity data).

(BTW, I was never fully convinced that running a workflow against a snapshot was sufficiently different than running a workflow against entity data to warrant a separate test, though I'll admit that I never questioned it aloud or thought too hard about it.)

Reply:
I see - that makes sense.

I think I conflated two points of my argument.
1. The possibility of wanted to check, during a journey, that some functionality is working - like verifying user inputs before allowing them to proceed.
2. A code maintenance / dev ergonomics perspective (not really a "we need to test for this" concept). If a test fails one of the first steps I take is to run through the journey myself. While I do so I like to look at what the test is doing. If I were new the codebase I could see viewing a disabled button as part of that journey and thinking that is the issue. Seeing something in the test code as a hint that the test dev accounted for this may save me some time.

I am ok proceeding either way.

Reply:
Unfortunately disabled always present is causing a number of test failures. I suspect anchor tags, as I really wouldn't expect aria-disabled on either anchor or button elements (although we do support setting aria-disabled to true on an anchor tag).

I really am having trouble feeling excited about figuring out an xpath that will work for all of our potentially clickable items. I personally would rather kick that down the road until it is truly needed.

Reply:
It turns out that there are a lot of times when we will not have the `aria-disabled` property present, even on things with role="button". Therefore I'm going back to the explicit call with the expectation that it is only used when you know that the element you are waiting on has the expectation of being initially disabled.

Reply:
Yes, exactly. We have to take into account firecloud UI "buttons" (method repository), which are anchor tags without aria, and "Save cohort" which is an actual button, and I don't know what else. And then we have to look for the absence of attributes/properties that would indicate disabled. I think this is out-of-scope at this point.